manifests/crds/network.harvesterhci.io_vlanconfigs.yaml

```diff
@@ -57,11 +57,14 @@ spec:
               uplink:
                 properties:
                   bondOptions:
-                    description: 'TODO: reference: https://www.kernel.org/doc/Documentation/networking/bonding.txt'
+                    description: 'reference: https://www.kernel.org/doc/Documentation/networking/bonding.txt'
                     properties:
                       miimon:
+                        default: -1
```

Comment:
Question:
why the default value of miimon is -1, default value in https://www.kernel.org/doc/Documentation/networking/bonding.txt is 0

Reply:
`keep the original value` should be `keep the existing value`?